corehq/apps/user_importer/helpers.py

```diff
@@ -5,4 +5,7 @@ def spec_value_to_boolean_or_none(user_spec_dict, key):
     value = user_spec_dict.get(key, None)
     if value and isinstance(value, str):
         return string_to_boolean(value)
-    return value
+    elif isinstance(value, bool):
+        return value
+    else:
+        return None
```

Comment:
the last else block looks redundant.

Reply:
Good point, though in this case I think it's actually maybe better to be explicit. Do you disagree?

Reply:
Being explicit is great and i am always in favour of it, just that here it's in the end of the method and would be clear without the else as well.
There is also an option to entirely skip `return None` which would make this implicit, which i am not asking for here.

Reply:
(fine to punt)